submissions/MarharytaBoichenko/dom_menu/js/refs.js

```diff
@@ -0,0 +1,4 @@
+const navList = document.querySelector(".nav__list");
```

Comment:
This file is pretty small, you could've placed those in the main file just fine.